ldap/servers/slapd/back-ldbm/ldbm_modrdn.c

```diff
@@ -1400,9 +1406,10 @@ ldbm_back_modrdn(Slapi_PBlock *pb)
                     slapi_pblock_set(pb, SLAPI_PLUGIN_OPRETURN, ldap_result_code ? &ldap_result_code : &retval);
                 }
                 slapi_pblock_get(pb, SLAPI_PB_RESULT_TEXT, &ldap_result_message);
+                betxn_callback_fails = 1;
```

Comment:
Seems like an overkill:
  Why setting a variable to True then testing it just after rather than keeping the old code ?